src/odroid.md

```diff
@@ -19,6 +19,11 @@
         time.sleep(0.6)
     ```
 
+Once assembled it should look like so:
+![Power board](images/power_board.jpg)
```

Comment:
Nice!
Any chance we have an image which uses an ODROID connector cable which has the sheath properly seated (covering the wires), rather than one which is damaged?